backend/users/models.py

```diff
@@ -16,3 +17,47 @@ class CustomUser(AbstractUser):
     token_version = models.IntegerField(default=0)
     is_authenticated = models.BooleanField(default=False)
     is_42 = models.BooleanField(default=False)
+    last_activity = models.DateTimeField(default=timezone.now)
+    friends = models.ManyToManyField(
```

Comment:
could you explain how this relationship works so i have a better idea of this

Reply:
so every User will have `friends` list, but to define the relation between two friends i am using "Friendship" class. So every User will have list of Friendships that will be defined as Friendship object/model

Reply:
that makes sense, so this way one user can have multiple friendship objects and then refer to their friends